### DISCUSSION TITLE
How to Embed a Private Video

### DISCUSSION THREAD

**Discussion by jeremychr** [mediacms-io/mediacms#987] [Q&A]
I want to embed a private video, but there is no clue where should i give my authentication in the embed code (eg. src="https://demo.mediacms.io/embed?m=XItrWzzp4").
I don't want make the video unlisted, because i very need it to be private. So, don't give advice to change my video state.
Thanks for any replies.

**Comment by jeremychr** [comment]
Is there any way to do it like this: https://username:password@demo.mediacms.io/embed?m=friendly_token
Any response is appreciate, Thanks.

**Comment by Freigeist069** [comment]
Hi there,

maybe [this post](https://github.com/mediacms-io/mediacms/discussions/674) helps you.

Best, Joerg
